Epsilon.Canvas.Abstractions/Rest/Page.cs

```diff
@@ -31,13 +31,13 @@ public Page(string id)
     public string? CreatedAt { get; init; }
 
     [JsonPropertyName("url")]
-    public string? Url { get; init; }
+    private string? Url { get; }
```

Comment:
I don't think these should be private, could you give a reason?

Reply:
Fair, But for now it is only used within the class. So this is an option or suppress the notice.